boa/src/builtins/property/attribute/mod.rs

```diff
@@ -20,6 +20,9 @@ bitflags! {
         /// None of the flags are present.
         const NONE = 0b0000_0000;
 
+        /// All the flags set (`WRITABLE`, `ENUMERABLE`, `CONFIGURABLE`).
+        const ALL = 0b0011_1111;
+
```

Comment:
If I'm not mistaken, we can use the `Attribute::empty()` and `Attribute::all()` for this, right? So we don't need these two.

[More information](https://docs.rs/bitflags/1.2.1/bitflags/example_generated/struct.Flags.html)